tools/mozdevice/logging.js

```diff
@@ -107,6 +107,15 @@ Logging.prototype.error = function(tag, message) {
  */
 Logging.prototype.clear = function() {
   debug('Clearing');
+  if (this.device.properties['ro.build.version.sdk'] >= 21) {
+    var logging = this;
+    return new Command()
+      .append('date +"%m-%d %H:%M:%S.000"')
```

Comment:
I think you should be able to insert this command directly via `new Command(datecommand)`.